### PR TITLE
React 2 Week1 Homework

### PR DESCRIPTION
**Pull request by na-talia**

### CODE REVIEW

File: react/react2/week1/github-search/src/components/UserContext.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    const API_URL = `https://api.github.com/search/users?q=${query}`;
+
+    try {
+      setLoading(true);
+      const response = await fetch(API_URL);
+      const data = await response.json();
+
+      data.items
+        ? setUsers(data.items) && setError(false)
+        : setUsers([]) && setError("No results");
```
Comment (Divyajg) [inline react/react2/week1/github-search/src/components/UserContext.js]: You are handling the no-results situation in the searchUser.js. No need for it here.

File: react/react2/week1/github-search/src/components/UserContext.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    } catch (error) {
+      setError(error.message);
+    }
+    setLoading(false);
+  };
```
Comment (Divyajg) [inline react/react2/week1/github-search/src/components/UserContext.js:26]: You can use `finally()` after `catch()`. [Docs for your reference](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/try...catch)

**Review by Divyajg — APPROVED** [review]
Good job 👍

**Comment by na-talia** [comment]
> Good job 👍

Thank you so much! I'll fix the code.

**Pull request merged**